pkg/ovs/ovsctl/appctl.go

```diff
@@ -227,3 +228,49 @@ func (c *ovsCtlClient) GetDPFeatures() (map[DPFeature]bool, error) {
 	}
 	return features, nil
 }
+
+// DeleteDPInterface deletes OVS datapath interface, and it returns with no error if the interface does not exist.
+func (c *ovsCtlClient) DeleteDPInterface(name string) error {
+	cmd := fmt.Sprintf("dpctl/show ovs-system")
+	out, err := c.runAppCtl(cmd, true)
+	if err == nil {
+		return nil
+	}
+	scanner := bufio.NewScanner(strings.NewReader(string(out)))
+	scanner.Split(bufio.ScanLines)
+	ports := make([]int, 0)
+	for scanner.Scan() {
+		line := scanner.Text()
+		fields := strings.Split(line, ": ")
+		if len(fields) != 2 {
+			klog.InfoS("Unexpected output from dpif/show ovs-system", "line", line)
```

Comment:
this may mismatch other cases, for example `port 1: genev_sys_6081 (geneve: packet_type=ptap)`. Although ExternalNode doesn't have such port, this is a generic function so should be more precise.
```
system@ovs-system:
  lookups: hit:188080 missed:6956 lost:0
  flows: 0
  masks: hit:1111386 total:0 hit/pkt:5.70
  port 0: ovs-system (internal)
  port 1: genev_sys_6081 (geneve: packet_type=ptap)
  port 2: antrea-gw0 (internal)
  port 3: nginx-76-8686ea
  port 4: nginx-76-92d4d0
```

Reply:
Thanks for pointing out. I have changed this part